### DISCUSSION TITLE
How to add new search engine?

### DISCUSSION THREAD

**Discussion by M1sterXXL** [bromite/bromite#1432] [Q&A]
i want to add startpage.help pls

**Comment by sphinxcdi** [comment]
https://github.com/bromite/bromite/wiki/SearchEngines

For Startpage to show up as an option:
1. go to [their settings](https://www.startpage.com/do/settings#privacy_and_safety)
2. set 'HTTP request method' to `GET`
3. save your settings
4. proceed with https://github.com/bromite/bromite/wiki/SearchEngines#adding-a-new-search-engine

**Reply by M1sterXXL** [reply]
thx!
